drivers/serial/uart_realtek_rts5912.c

```diff
@@ -50,11 +134,85 @@ static int rts5912_uart_init(const struct device *dev)
 		return rc;
 	}
 
+#if defined(CONFIG_PM)
+	const uint32_t uart_reg = uart_ns16550_get_port(dev_cfg->uart_dev);
+	uint32_t iir = sys_read32(ADDR_IIR) & IIR_MASK;
+	uint32_t _wf_cycle_count = k_us_to_cyc_ceil32(RTS5912_MAXIMUM_UART_POLLING_TIME_US);
```

Comment:
Non-blocking comment, the leading underscores for the local variable names is not needed.